syntax/asciidoctor.vim

```diff
@@ -316,6 +320,11 @@ else
     hi def asciidoctorBoldItalic             gui=bold,italic cterm=bold,italic
 endif
 
+hi def asciidoctorHighlight                 guibg=gold1 ctermbg=3 term=reverse
```

Comment:
This has to be either linked to one of existing default highlight groups or left undefined. Let end user define it according to their colorscheme.